src/cozmo/robot.py

```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
 
         self._action_dispatcher = self._action_dispatcher_factory(self)
 
+        #: :class:`cozmo.util.Speed` Speed of the left wheel
         self.left_wheel_speed = None
+        #: :class:`cozmo.util.Speed` Speed of the right wheel
         self.right_wheel_speed = None
-        self.light_height = None
+        #: :class:`cozmo.util.Distance` height of the lift from the ground (in :const:`MIN_LIFT_HEIGHT_MM` to :const:`MAX_LIFT_HEIGHT_MM` range)
```

Comment:
Each of these descriptions should start with a capital letter to match the existing descriptions.